### DISCUSSION TITLE
How do I use "match" for macOS Installer cert?

### DISCUSSION THREAD

**Discussion by BirjuVachhani** [fastlane/fastlane#21188] [Q&A] [answered]
Hello,

I am trying to write a fastlane script to automate macOS deployment. I am using `match` to store my certs and provisioning profiles in a private repo. How do I get `match` to generate certs for mac installer distribution?

**Comment by alebx** [comment, marked as answer]
You have to pass the `--additional_cert_types mac_installer_distribution` parameter.

**Reply by BirjuVachhani** [reply]
@alebx Thank you for your answer. I was aware of this option but I only used it to create certificates with match command, I wasn't using it in the fastlane lane actually because I though this was just a command line option or a Matchfile option. I am using it in fastfile now and it works!
